src/Chapter2.hs

```diff
@@ -620,7 +629,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven [x] = [x]
+takeEven (x : _ : xs) = x : takeEven xs
```

Comment:
Wonderful pattern-matching skills 👏🏼

Reply:
To be honest, not very clear about when to use [...] vs (...) syntax while using lists in pattern matching. Had to use a trial-and-error method with Task 5: `isThird42` as well...